docker/docker-compose.ci.mysql.yml

```diff
@@ -3,7 +3,7 @@ version: '3.8'
 services:
   killbill:
     network_mode: host
-    image: killbill/killbill:0.22.20
+    image: killbill/killbill:0.24.0
```

Comment:
Would it make sense to use `latest` - so we get most recent of `0.24.x` ?

Reply:
I'd rather have something _stable_ (predictable) here. Also, `latest` takes a while to boot, so it would slow down tests.